apps/client/src/services/utils.ts

```diff
@@ -124,8 +124,31 @@ function formatDateISO(date: Date) {
     return `${date.getFullYear()}-${padNum(date.getMonth() + 1)}-${padNum(date.getDate())}`;
 }
 
-function formatDateTime(date: Date) {
-    return `${formatDate(date)} ${formatTime(date)}`;
+
+// old version
```

Comment:
I would remove the whole "old version" block, instead of keeping it around in an commented out state.
If you want to check out an older version -> we have git for that ;-)